new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItDomainInPV.java

```diff
@@ -764,236 +730,29 @@ private void createDomainOnPVUsingWdt(Path domainCreationScriptFile, Path modelF
-      logger.severe(ioex.getMessage());
-      fail("Create persistent volume host path failed");
-    }
-
-    V1PersistentVolume v1pv = new V1PersistentVolume()
-        .spec(new V1PersistentVolumeSpec()
-            .addAccessModesItem("ReadWriteMany")
-            .storageClassName("weblogic-domain-storage-class")
-            .volumeMode("Filesystem")
-            .putCapacityItem("storage", Quantity.fromString("5Gi"))
-            .persistentVolumeReclaimPolicy("Recycle")
-            .accessModes(Arrays.asList("ReadWriteMany"))
-            .hostPath(new V1HostPathVolumeSource()
-                .path(pvHostPath.toString())))
-        .metadata(new V1ObjectMeta()
-            .name(pvName)
-            .putLabelsItem("weblogic.domainUid", domainUid));
-    boolean success = assertDoesNotThrow(() -> createPersistentVolume(v1pv),
-        "Failed to create persistent volume");
-    assertTrue(success, "PersistentVolume creation failed");
-  }
-
-  /**
-   * Create a persistent volume claim.
-   *
-   * @param pvName name of the persistent volume
-   * @param pvcName name of the persistent volume to create
-   * @param domainUid UID of the WebLogic domain
-   * @param namespace name of the namespace in which to create the persistent volume claim
-   */
-  private void createPVC(String pvName, String pvcName, String domainUid, String namespace) {
-    logger.info("creating persistent volume claim");
-
-    V1PersistentVolumeClaim v1pvc = new V1PersistentVolumeClaim()
-        .spec(new V1PersistentVolumeClaimSpec()
-            .addAccessModesItem("ReadWriteMany")
-            .storageClassName("weblogic-domain-storage-class")
-            .volumeName(pvName)
-            .resources(new V1ResourceRequirements()
-                .putRequestsItem("storage", Quantity.fromString("5Gi"))))
-        .metadata(new V1ObjectMeta()
-            .name(pvcName)
-            .namespace(namespace)
-            .putLabelsItem("weblogic.domainUid", domainUid));
-
-    boolean success = assertDoesNotThrow(() -> createPersistentVolumeClaim(v1pvc),
-        "Failed to create persistent volume claim");
-    assertTrue(success, "PersistentVolumeClaim creation failed");
-  }
-
   /**
    * Create secret for docker credentials.
    *
    * @param namespace name of the namespace in which to create secret
    */
   private void createOCRRepoSecret(String namespace) {
-    CommonTestUtils.createDockerRegistrySecret(OCR_USERNAME, OCR_PASSWORD,
+    createDockerRegistrySecret(OCR_USERNAME, OCR_PASSWORD,
         OCR_EMAIL, OCR_REGISTRY, OCR_SECRET_NAME, namespace);
   }
 
   private File createScriptToKillServer() throws IOException {
     File killServerScript = File.createTempFile("killserver", ".sh");
     //deletes the file when VM terminates
     killServerScript.deleteOnExit();
-    FileWriter fw = new FileWriter(killServerScript);
-    fw.write("#!/bin/bash\n");
-    fw.write("kill -9 `jps | grep Server | awk '{print $1}'`");
-    fw.close();
```

Comment:
fw.close() call is missing in the refactored code. Not sure if it is intentional.

Reply:
Yes that is intentional. The try-with-resource will auto close the resource regardless of if there is a exception thrown or not.